MAUI/Autocomplete/Highlighting-matched-text.md

```diff
@@ -0,0 +1,97 @@
+---
+layout: post
+title: Highlighting matched text in AutoComplete control | Syncfusion
+description: Learn about Highlighting matched text in .NET MAUI SfAutoComplete (SfAutocomplete) control and more details.
+platform: maui
+control: SfAutocomplete
+documentation: ug
+---
+
+# Highlighting matched text
+
+## Highlighting matched text in .NET MAUI SfAutoComplete
+
+Highlight matching characters in a suggestion list to pick an item with more clarity. There are two ways to highlight the matching text:
+
+*   `First Occurrence`
+
+*   `Multiple Occurrence`
+
+The text highlight can be indicated with various customizing styles by enabling the below properties. They are
+
+*   `HighlightedTextColor` - sets the color of the highlighted text for differentiating the highlighted characters.
+
+*   `HighlightTextFontAttributes` - sets the FontAttributes of the highlighted text.
+
+## First Occurrence
+It highlights the first position of the matching characters in the suggestion list.
+
+{% tabs %}
+
+{% highlight xaml %}
+
+ <editors:SfAutocomplete x:Name="autocomplete"
+      WidthRequest="300"
+      ItemsSource="{Binding SocialMedias}"
+      TextHighlightMode="FirstOccurrence"
+      TextMemberPath="Name"
+      DisplayMemberPath="Name"
+      HighlightedTextColor="Red"
+      HighlightedTextFontAttributes="Bold"
+      TextSearchMode="StartsWith" />
+
+{% endhighlight %}
+
+{% highlight c# %}
+
+SfAutocomplete autoComplete = new SfAutocomplete() 
+{
+ HeightRequest = 40,
+ TextSearchMode = AutocompleteTextSearchMode.StartsWith,
```

Comment:
By default, the text search mode is StartsWith. So, remove it.

Reply:
Updated changes